public/robots.txt

```diff
@@ -2,6 +2,8 @@
 #
 # To ban all spiders from the entire site uncomment the next two lines:
  User-agent: *
- Disallow: /flags
+ Disallow: /admins
+ Disallow: /blockeds
+ Disallow: /activities
 
  Sitemap: http://www.catforum.mntc.in/sitemap.xml
```

Comment:
ok then can i do it in this PR? yeah there isn't.but anyways they won't be able to see since activities need a current user without which it will generate an error and bockeds url is also generating some error.actually blockeds is not serving any business.all block activities is controlled by flags controller.